docs/tutorials/task.cis-to-component-definition/transformation.md

```diff
@@ -0,0 +1,1092 @@
+# Tutorial: Setup for and use of ComplianceAsCode profile to OSCAL Component Definition transformer
+
+Here are step by step instructions for setup and transformation of [ComplianceAsCode](https://github.com/ComplianceAsCode/content) profile data files into [NIST](https://www.nist.gov/) standard [OSCAL](https://pages.nist.gov/OSCAL/) [Component Definition](https://pages.nist.gov/OSCAL/reference/latest/component-definition/json-outline/) using the [compliance-trestle](https://ibm.github.io/compliance-trestle/) tool.
+
+## *Objective*
+
+How to transform one or more `.profile` compliance files into a standardized `OSCAL.json` file.
+
+There are 2 short steps shown below.
+The first is a one-time check/set-up of your environment.
+The second is a one-command transformation from `.profile` to `OSCAL.json`.
+
+## *Step 1: Environment setup*
+
+Linux, Mac
+
+<details>
+<summary>Windows</summary>
+
```

Comment:
You could just provide a link to the generic trestle setup tutorial - or provide this concise version but point to it for more details.  https://ibm.github.io/compliance-trestle/python_trestle_setup/

Reply:
Agreed. I will use just the link.